_includes/references.md

```diff
@@ -117,11 +117,12 @@
 [BIP174]: https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki
 
 [@bitcoincoreorg]: https://twitter.com/bitcoincoreorg
-[#bitcoin-core-dev]: https://webchat.freenode.net?channels=%23bitcoin-core-dev&uio=MTY9dHJ1ZSYxMT0yMTU87
+[#bitcoin-core-dev]: https://webchat.freenode.net?channels=%23bitcoin-core-dev
+[#bitcoin-builds]: https://webchat.freenode.net?channels=%23bitcoin-builds
+[##bitcoin-builds]: https://webchat.freenode.net?channels=%23%23bitcoin-core-gui
```

Comment:
I think there's typos in the two lines above: links named both `#bitcoin-builds` (one hash) and `##bitcoin-builds` (two hashes), and one `-builds` link points to the `bitcoin-core-gui` channel.

Reply:
you're right, will fix